service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsControllerTest.java

```diff
@@ -70,28 +78,76 @@ void shouldNotReturnAnErrorWhenHearingDateIsSetInDistantFuture() throws Exceptio
         LocalDateTime distantFuture = LocalDateTime.now().plusYears(1000);
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(createHearing(
-            distantFuture, distantFuture.plusDays(1)));
+            distantFuture, distantFuture.plusDays(1)), EMPTY_BOOKING);
+
+        assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
+    }
+
+    @Test
+    void shouldOnlyValidateNewBookingsWhenBookingAlreadyExists() throws Exception {
+        LocalDateTime distantFuture = LocalDateTime.now().plusYears(1000);
+        LocalDateTime distantPast = LocalDateTime.now().minusYears(10000);
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(createHearing(
+            distantFuture, distantFuture.plusDays(1)), createHearing(distantPast, distantPast.plusDays(1)));
```

Comment:
That test is not really testing if the old hearing is skipped does it?

Correct me if I am wrong, but that request is essentially removing the distantHearing as the "after" portion contains only one hearing.

Reply:
nope you're right. Different ids because of wrap elements, shall change